### ISSUE TITLE
Imports from `.` no longer working after upgrading >= 0.13.0

### ISSUE THREAD

**Issue by asottile** [dahlia/libsass-python#208]
Discovered in https://github.com/pre-commit/pre-commit.github.io/pull/113

Here's a simple test case:

```console
$ tail -n999 scss/*
==> scss/main.scss <==
@import 'scss/variables';

==> scss/_variables.scss <==
```

Before:

```console
$ ~/workspace/pre-commit.github.io/venv/bin/sassc --version
sassc 0.12.3 (sass/libsass 3.4.3)
$ ~/workspace/pre-commit.github.io/venv/bin/sassc scss/main.scss out/out.css
$ echo $?
0
```

After:

```console
$ ~/workspace/pre-commit.github.io/venv/bin/sassc --version
sassc 0.13.0 (sass/libsass 3.4.4)
$ ~/workspace/pre-commit.github.io/venv/bin/sassc scss/main.scss out/out.css
sassc: error: Error: File to import not found or unreadable: scss/variables.
       Parent style sheet: /tmp/foo/scss/main.scss
        on line 1 of scss/main.scss
>> @import 'scss/variables';
   ^

```

**Comment by asottile** [comment]
I'm manually bisecting, my first *guesses* at what changed / broke this were incorrect... I may resort to actual bisection :)

**Comment by asottile** [comment]
Bumping the submodule back to 3.4.3 fixes it, so it seems to be some behaviour changed between 3.4.3 and 3.4.4

The changelog doesn't seem to indicate anything interesting: https://github.com/sass/libsass/releases/tag/3.4.4

**Comment by asottile** [comment]
For now, resolved via #209

**Comment by asottile** [comment]
The fix for this is in 0.13.2
